packages/@adobe/spectrum-css-temp/components/search/index.css

```diff
@@ -74,6 +78,10 @@ governing permissions and limitations under the License.
   .spectrum-Textfield.spectrum-Search.is-valid & {
     padding-inline-end: calc(var(--spectrum-search-padding-right) + var(--spectrum-textfield-padding-x) + var(--spectrum-icon-checkmark-medium-width) + var(--spectrum-textfield-icon-margin-left));
   }
+
+  .spectrum-Textfield.spectrum-Search.is-loading & {
```

Comment:
I think these class names `is-invalid` `is-valid` were added recently, all of them should probably conform to the state classname pattern though
`spectrum-Search--isInvalid`
`spectrum-Search--isValid`
`spectrum-Search--isLoading`

Reply:
for reference, see quiet https://github.com/adobe/react-spectrum/pull/1335/files#diff-cd526b7742dcb468383ec2ee115e3ed927d5100519b2e7d6e5a730b6968a04f9R222
`'spectrum-Textfield--quiet': isQuiet`
whether we keep the `is` in front doesn't matter to me, it's state, so it'll be read as a boolean anyways

Reply:
Sounds good, I'll change the combobox/textfield specific patterns in this pull and open an issue to do the same for things like stepper, checkbox, etc